Platform/AlderlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c

```diff
@@ -1096,7 +1096,6 @@ UpdateFspConfig (
     FspsConfig->SaPcieItbtRpSnoopLatencyOverrideValue[2] = 0xc8;
     FspsConfig->TdcTimeWindow[0] = 0x3e8;
     FspsConfig->TdcTimeWindow[1] = 0x3e8;
-    FspsConfig->PchLockDownBiosLock = 0x1;
```

Comment:
Can you please add this at the later stage? It is probably applicable to all ADL series.

Reply:
Do you mean at the end of OS loader? Looking at the current ADL BIOS code, it appears that this value is defaulted to 0x0. It also appears that this setting performs the same operation as ProgramSecuritySetting(), just within FSP-S. We call ProgramSecuritySetting() at end of OS loader for OS loader payload. For all other non-firmware update payloads, we call ProgramSecuritySetting() at the end of stage 2. Having this value set to 0x1 prevents the tool from writing to flash.

Reply:
Ok makes sense. Thanks for the details.